pymc3/step_methods/metropolis.py

```diff
@@ -144,6 +143,19 @@ def __init__(
         mode=None,
         **kwargs
     ):
+        """Create an instance of a Metropolis stepper
+
+        Arguments:
+        vars -- List of variables for sampler (default None)
```

Comment:
This looks very different from all our other doc strings which follow the numpy style guide, can you make it adhere to that?

Reply:
Oh Sorry! My bad. I will fix it right away

Reply:
Done @twiecki : )